apps/console/src/config/theme.ts

```diff
@@ -16,11 +16,10 @@
  * under the License.
  */
 
-// eslint-disable-next-line no-restricted-imports
-import { Theme } from "@oxygen-ui/react";
+import { Theme } from "@oxygen-ui/react/models";
 import { extendTheme } from "@oxygen-ui/react/theme";
 
-export const AsgardeoTheme: Theme = extendTheme({
+export const ConsoleTheme: Theme = extendTheme({
```

Comment:
Let's just go with `Theme`.
cc:/ @jathushan-r 

```suggestion
export const Theme: Theme = extendTheme({
```

Reply:
Ack.

Reply:
Addressed with a3a4245